hnn_core/tests/test_cell_response.py

```diff
@@ -24,7 +24,9 @@ def test_cell_response(tmp_path):
                                  spike_gids=spike_gids,
                                  spike_types=spike_types,
                                  times=sim_times)
-    cell_response.plot_spikes_hist(show=False)
+    kwargs_hist = dict(linewidth=1.2,
+                       histtype="step")
+    cell_response.plot_spikes_hist(show=False, **kwargs_hist)
```

Comment:
this is just a smoke test ... any way we can check if the new linewidth is reflected in the matplotlib object properties?

Reply:
Hi , my apologies for the confusion... linewidth isn't a parameter in hist plots. heres how histtype changes reflect
![image](https://github.com/jonescompneurolab/hnn-core/assets/94792103/cd6e4880-e005-445f-8d49-6f971be714d1)

Reply:
indeed but your test doesn't check that it changed in someway ... you need to do something like:

```py
fig = cell_response.plot_spikes_hist(show=False, **kwargs_hist)
```

and then:

```py
assert fig.some_attribute = expected_attribute_value
```

Reply:
After trying a couple of techniques to ensure that the value passed by `kwargs_hist` is returned, I can assert whether the alpha value was indeed passed or not. 
This covers the added feature.